site/src/pages/api/blocks.api.ts

```diff
@@ -1,10 +1,16 @@
+import Ajv from "ajv";
 import { query as queryValidator } from "express-validator";
+import { cloneDeep } from "lodash";
 import blocksData from "../../../blocks-data.json";
-import { ExpandedBlockMetadata as BlockMetadata } from "../../lib/blocks";
+import {
+  ExpandedBlockMetadata as BlockMetadata,
+  readBlockDataFromDisk,
+} from "../../lib/blocks";
 import { createApiKeyRequiredHandler } from "../../lib/handler/apiKeyRequiredHandler";
 
 export type ApiSearchRequestQuery = {
   q: string;
+  json: string;
```

Comment:
These should be optional, so that users can provide either (or neither, for a full list - later, paginated)

```suggestion
  q?: string;
  json?: string;
```